src/lxc/terminal.c

```diff
@@ -933,10 +932,6 @@ int lxc_terminal_setup(struct lxc_conf *conf)
 	if (ret < 0)
 		return -1;
 
-	ret = lxc_setup_tios(terminal->master, &oldtios);
```

Comment:
Yeah, so that one is one me. I missed the point that we want to set tios properties on the proxy pty we are using not on the actual pty that we attach to. So the change in the commit you're referencing should actually only remove the `\n\r` property from the tios setup but not muck with the actual terminal setup. Neither attach nor console should have the terminal properties changed .

Reply:
This is the same behavior as in LXD.

Reply:
thanks